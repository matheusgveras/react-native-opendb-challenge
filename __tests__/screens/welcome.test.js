import 'react-native';
import React from 'react';
import Welcome from '../../src/screens/welcome';
import renderer from 'react-test-renderer';
describe('testing screen welcome', () => {
  it('renders welcome correctly', () => {
    const tree = renderer.create(<Welcome />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});