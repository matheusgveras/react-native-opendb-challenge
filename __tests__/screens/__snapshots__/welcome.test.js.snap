// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders welcome correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/logo.png",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "height": 150,
          "width": 150,
        }
      }
    />
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/logo.png",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "height": 55,
          "width": 150,
        }
      }
    />
  </View>
  <View>
    <Text>
      teste
    </Text>
  </View>
</View>
`;

exports[`testing screen welcome renders welcome correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#cccccc",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/logo.png",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "height": 150,
          "width": 150,
        }
      }
    />
    <Image
      source={
        Object {
          "testUri": "../../../src/resources/images/logo.png",
        }
      }
      style={
        Object {
          "alignSelf": "center",
          "height": 55,
          "width": 150,
        }
      }
    />
  </View>
  <View>
    <Text>
      teste
    </Text>
  </View>
</View>
`;
